src/github/graphql.ts

```diff
@@ -15,11 +16,14 @@ export interface GraphQLError {
 }
 
 export function addGraphQL (client: GitHubAPI) {
-  const graphql = octokitGraphqlWithDefaults(client.request, {
-    method: 'POST',
```

Comment:
Removed superfluous `method` and `url` properties. They are already set in https://github.com/octokit/graphql.js/blob/d457a93b87cdc233bce1e8a0b7149b850fdb660e/src/index.ts#L18